### ISSUE TITLE
Rotating ImageMobject in 3D is buggy

### ISSUE THREAD

**Issue by sparshg** [ManimCommunity/manim#3131]
## Description of bug / unexpected behavior

In the animation below where image 'lies' on `NumberPlane`, It always remains a rectangle instead of getting distorted into a trapezium.
However, doing same with opengl renderer works as expected.

## Bug Animation

<details><summary>Images/GIFs</summary>

https://user-images.githubusercontent.com/43041139/213927772-1d23528e-5894-4156-98dd-16d992820e47.mp4

</details>

## Code

<details><summary>Code</summary>

```py
num = NumberPlane()
img = ImageMobject("test.png")
self.add(num, img)
self.play(Rotate(num, -PI / 2, axis=RIGHT), Rotate(img, -PI / 2, axis=RIGHT))
```

</details>

## Logs
<details><summary>Terminal output</summary>

```
PASTE HERE OR PROVIDE LINK TO https://pastebin.com/ OR SIMILAR
```

</details>

## System specifications

<details><summary>System Details</summary>

- OS (with version, e.g., Windows 10 v2004 or macOS 10.15 (Catalina)):
- RAM:
- Python version (`python/py/python3 --version`):
- Installed modules (provide output from `pip list`):
```
PASTE HERE
```
</details>

<details><summary>LaTeX details</summary>

+ LaTeX distribution (e.g. TeX Live 2020):
+ Installed LaTeX packages:

</details>

<details><summary>FFMPEG</summary>

Output of `ffmpeg -version`:

```
PASTE HERE
```
</details>

**Comment by kevinlinxc** [comment]
This also happens with the camera orientation:

https://github.com/ManimCommunity/manim/assets/44324811/f7f036b8-7356-4803-ab35-9b7aa2eebf3e


```python
from manim import *

class MyScene(ThreeDScene):
    def construct(self):
        self.video1 = ImageMobject(
                filename_or_array="cat.png",
            ).scale_to_fit_height(3)
        ax = Axes(
            x_range=[0, 10, 1],
            x_length=9,
            y_range=[0, 20, 5],
            y_length=6,
            axis_config={"include_numbers": True, "include_tip": False},

        ).to_edge(DL + RIGHT + UP, buff=1).scale(0.7)
        labels = ax.get_axis_labels()

        self.play(Create(VGroup(ax, labels)))
        self.play(FadeIn(self.video1))
        self.wait(3)
        self.move_camera(phi=0*DEGREES, theta= -90 * DEGREES, zoom= 0.7, run_time=0.4, gamma=0*DEGREES)

        self.begin_ambient_camera_rotation(90 * DEGREES / 3, about='phi')
        self.begin_ambient_camera_rotation(90 * DEGREES / 3, about='theta')
        self.wait(3)
```

**Comment by kleinesfilmroellchen** [comment]
I banged my head against this for several hours, but I figured something out that may be worth investigating:

Pillow's Image has a `transform` method that can do a perspective transform. The parameter appears to just be the homogeneous 3D matrix, so it *should* be possible to use the inverse of the rotation matrix shifted to the center of the image. (`inverse(translation) * rotation * translation`). I couldn't get this to work however.

**Comment by uwezi** [comment]
I don't think you want to mess around with the pixel matrix itself. The Manim renderer should correctly find the pixels in the image matrix and calculate its position on the projected camera's focus plane.
